crawl-ref/source/art-func.h

```diff
@@ -199,16 +204,16 @@ static bool _DISPATER_targeted_evoke(item_def */*item*/, bool* did_work, bool* u
     }
 
     *did_work = true;
-    dec_hp(hp_cost, false);
-    dec_mp(mp_cost);
+    you.hp -= hp_cost;
```

Comment:
I'm fine with adding an accessor, but it'd just wrap exactly this. I do *not* want to use the dec accessor (at least without a flag), since that takes logfile notes and prints warnings, which we don't want to do while we're still in a targeter!